site/docs/latest/admin-api/persistent-topics.md

```diff
@@ -644,3 +644,10 @@ String topic = "persistent://my-tenant/my-namespace/my-topic";
 String subscriptionName = "my-subscription";
 admin.persistentTopics().deleteSubscription(topic, subscriptionName);
 ```
+
+### Last Message Id
+
+It gives the last commited message ID for a persistent topic.
+
+#### REST API
```

Comment:
typically for a given rest endpoint, it should be documented for 3 parts, 1) http rest api 2) pulsar-admin 3) admin java api. Can you follow what other endpoints do?

Also did we expose the endpoint to admin java api?